src/ExerciseRunner/CgiRunner.php

```diff
@@ -145,6 +166,44 @@ private function checkRequest(RequestInterface $request, string $fileName): CgiR
         return new Success($request);
     }
 
+    private function setupEnvironment(ExecutionContext $context, SolutionInterface $solution): void
+    {
+        $filesystem = new Filesystem();
+
+        foreach ($solution->getFiles() as $file) {
+            $filesystem->copy(
+                $file->getAbsolutePath(),
+                Path::join($context->referenceEnvironment->workingDirectory, $file->getRelativePath()))
+            ;
+        }
+
+        foreach ($context->getFiles() as $fileName => $content) {
+            file_put_contents(
+                Path::join($context->referenceEnvironment->workingDirectory, $fileName),
+                $content
+            );
+        }
+//       sleep(1);
```

Comment:
docker tests fucked without this:

```
Error: "Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /host_mnt/private/var/folders/gg/rgfshctn263cdzq1zhn5y9300000gn/T/php-school/3d745671"
```

files must take a little while to sync 🤷